lgc/builder/BuilderImpl.cpp

```diff
@@ -688,109 +741,95 @@ Instruction *BuilderImpl::createWaterfallLoop(Instruction *nonUniformInst, Array
   SmallVector<Value *, 2> nonUniformIndices;
   // Maps the nonUniformIndex that is returned by traceNonUniformIndex() to the nonUniformInst.
   DenseMap<Value *, std::pair<Value *, unsigned>> nonUniformIndexImageCallOperand;
-  TraceNonUniformIndex traceNonUniformIndex(nonUniformInst, scalarizeDescriptorLoads, 64);
+  TraceNonUniformIndex traceNonUniformIndex(nonUniformInst, scalarizeDescriptorLoads);
+  DenseMap<unsigned, Value *> operandIdxnonUniformIndex;
```

Comment:
I think the naming convention is operandIdx**N**onUniformIndex, although I dislike having both `Idx` and `Index` at the same time.

Reply:
Done :)